build.gradle

```diff
@@ -39,8 +39,10 @@ ext {
     versionCode = 53010005
 
     minSdkVersion = 26
```

Comment:
The upstream repo has been using `minSdkVersion = 23`, is it a problem that we increase this here?

Reply:
Using 26 means there is no Java8 language feature desugaring (artifact transforms with dependencies), but using 26 sounds good as well.

Reply:
Went back to 23, not sure why I had problems before.